private/buf/cmd/buf/command/push/push.go

```diff
@@ -92,6 +93,7 @@ type flags struct {
 	CreateVisibility   string
 	CreateDefaultLabel string
 	SourceControlURL   string
+	SkipUnnamed        bool
```

Comment:
On second thought, I actually think we should call this `--exclude-unnamed` as we use `exclude` as the verb everywhere else.

Reply:
Renamed to exclude.